### ISSUE TITLE
Mirror Magnet feature: free movement of the camera frame during recording

### ISSUE THREAD

**Issue by RainerZufall9393** [appaheadgmbh/support#19]
A function in Mirror Magnet to move the camera frame freely during recording would be fantastic (as in Loom, for example).

**Comment by absoftware** [comment]
Yes, it would be nice feature. Plus from me.

But I would like to drag & drop from/to predefined areas to keep camera preview position always perfectly aligned. So aligning could be optional if someone prefers it exactly similar to Loom.

For me it would be perfect to drag & drop between these areas:

![Screenshot 2024-09-30 at 13 12 06](https://github.com/user-attachments/assets/037270b6-dc4d-4401-b109-7cb4a91afe16)

Adjusting camera preview size on the fly would be awesome as well. Loom gives very limited possibility to adjust size (small icon under the preview, it's visible only when mouse cursor is close enough).

**Comment by itsme-ken** [comment]
Yes, please! I have a 32-inch monitor and need to dedicate a 1920 x 1080 (16:9) recording area for my videos. To use Mirror Magnet effectively, I need the ability to freely move my camera preview into the specific recording area. My monitor is too large to record the entire screen—it wouldn’t look good in my videos. Being able to position the camera frame exactly where I want it would be very helpful.

![Image](https://github.com/user-attachments/assets/17d802b9-2112-4f78-8d73-1364f83709c0)
